pkgs/applications/science/logic/tlaplus/toolbox.nix

```diff
@@ -25,14 +25,16 @@ in stdenv.mkDerivation {
   inherit version;
   src = fetchzip {
     url = "https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-${version}-linux.gtk.${arch}.zip";
```

Comment:
```suggestion
    url = "https://tla.msr-inria.inria.fr/tlatoolbox/products/TLAToolbox-${version}-linux.gtk.x86_64.zip";
```

Reply:
Does this imply a more limited platform/architecture set? aarch64?

Reply:
Good catch, I have corrected the meta attribute to "x86_64-linux" as that tool has never been meant to be used on other platforms.